src/main/java/seedu/address/inventory/model/Item.java

```diff
@@ -146,8 +146,7 @@ public boolean isSameItem(Item otherItem) {
 
         return otherItem != null
                 && otherItem.getDescription().equalsIgnoreCase(getDescription())
-                && (otherItem.getCategory().equals(getCategory()) || otherItem.getCost() == (getCost())
-                || otherItem.getPrice() == getPrice());
+                && otherItem.getCategory().equals(getCategory());
```

Comment:
I thought the category should ignore cases too? I mean personally, I feel like there should not be different categories in different cases. Maybe you can change this?

Reply:
Oh thank you for pointing it out! I will change it immediately.